### PR TITLE
Dev modules, Popup additions

### PR DESCRIPTION
**Pull request by badgamernl**
#### Converted Health Indicator refactored:
- Made logic easier / less complex
- Added popup for player damage done to entities
- Changed module name to DamagePopup because felt more appropriate (feel free to change back)

### CODE REVIEW

File: modules/ChatPopup/softmod.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        "Chat",
+        "Popup",
+        "Mention",
+        "Floating",
+        "Text"
+    ],
+    "author": "badgamernl",
+    "contact": "badgamernl@gmail.com",
+    "license": "https://github.com/explosivegaming/scenario/blob/master/LICENSE",
+    "dependencies": {}
```
Comment (Cooldude2606) [inline modules/ChatPopup/softmod.json]: Dependencies need to be added.

File: modules/DamagePopup/softmod.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "keywords": [
+        "Damage",
+        "Popup",
+        "Floating",
+        "Text"
+    ],
+    "author": "badgamernl",
+    "contact": "badgamernl@gmail.com",
+    "license": "https://github.com/explosivegaming/scenario/blob/master/LICENSE",
+    "dependencies": {}
```
Comment (Cooldude2606) [inline modules/DamagePopup/softmod.json]: Dependencies need to be added.

File: modules/index.lua [hunk 3]
```diff
@@ -48,4 +48,6 @@ return {
     ['ExpGamingAdmin.Jail@4.0.0']='./modules/ExpGamingAdmin/Jail',
     ['ExpGamingCommands.admin@4.0.0']='./modules/ExpGamingCommands/admin',
     ['ExpGamingAdmin.Ban@4.0.0']='./modules/ExpGamingAdmin/Ban',
+    ['ChatPopup@4.0.0']='./modules/ChatPopup',
+    ['DamagePopup@4.0.0']='./modules/DamagePopup',
 }
```
Comment (Cooldude2606) [inline modules/index.lua]: In future note that the index can be remade using fsm install -d

**Comment by badgamernl** [comment]
done

**Pull request merged**